web/packages/core/src/ruffle-player.ts

```diff
@@ -148,7 +148,7 @@ export class RufflePlayer extends HTMLElement {
     private _readyState: ReadyState;
 
     private panicked = false;
-    private _cachedDebugInfo: string | null = null;
+    private debugInfo = "";
```

Comment:
Nit/question: Might it make sense to rename this to `rendererDebugInfo`? Or would it be better to keep its current name, in case other debug information than rendering is added to it later?

Reply:
Renamed to `rendererDebugInfo`.